### DISCUSSION TITLE
[ 功能建议 ]建议加个需求工单流程功能，在archery 上有个需求工单功能提工单走审批流程给DBA的话，达到统一管理的目的

### DISCUSSION THREAD

**Discussion by dengpixiong** [hhyo/Archery#1588] [Ideas]
### 目前遇到的问题/使用障碍
目前开发和运维遇到一些数据库方面的需求，如新增用户/数据库，变更用户/数据库，都是通过邮件的方式发给DBA，邮件往来不方便。

### 希望如何解决/实现它
在archery 上有个需求工单功能提工单走审批流程给DBA的话，达到统一管理的目的。

### 其他信息
如果有其他类似的产品功能或者图片信息，可在此提交

**Comment by LeoQuote** [comment]
没明白, 你意思是跟聊天一样, 就用文字描述自己需求吗?  如果是这样的话应该没有计划实现.

**Comment by dengpixiong** [comment]
类似下面的功能
![图片](https://user-images.githubusercontent.com/54564831/64008171-26c46600-cb48-11e9-88f8-1057e8bdc7f7.png)

**Comment by dengpixiong** [comment]
就是有个可以提需求工单的地方

**Comment by hhyo** [comment]
- 数据库创建可以直接在SQL上线提交，选择其他数据库即可，后面这个会适当优化，可以不选库
- 用户管理会逐步完善，做到工单化、自动化 #23 

单纯的描述工单暂不准备支持

**Comment by dengpixiong** [comment]
我个人建议后续可以考虑加上，我觉的archery定位为服务平台，而不只是工具，像阿里云，腾讯云帐号控制台右上角有个工单，很方便的提一些需求给后端人员。

**Comment by hhyo** [comment]
怎么说呢，这个想做也是一个比较系统化的模块，要考虑的比较多，否则单纯做一个记录的地方也好像比较鸡肋

- 工单分类
- 工单流转状态
- 工单各个状态的权限
- 通知逻辑

如果这几个能处理好也是挺好的，毕竟系统不可能将所有操作都做到标准化、自动化
